samcli/commands/publish/command.py

```diff
@@ -0,0 +1,138 @@
+"""CLI command for "publish" command."""
+
+import os
+import json
+import logging
+import click
+
+import boto3
+from botocore.exceptions import ClientError
+
```

Comment:
Keep imports to 3 sections, following PEP8

```
# Python standard library

# imports for dependencies

# imports for modules in the codebase
```

Reply:
Thanks, I fixed all the imports order.